packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -283,8 +287,13 @@ export class GCFBootstrapper {
   }
 
   gcf(
-    appFn: ApplicationFunction
+    appFn: ApplicationFunction,
+    wrapOptions?: WrapOptions
   ): (request: express.Request, response: express.Response) => Promise<void> {
+    let background = true;
+    if (wrapOptions) {
```

Comment:
🐛 - If I pass WrapOptions here as `{}`, or maybe with a second property down the road, it will set the default of wrapOptions to false, while it looks like you want the default to be true.

Instead I'd suggest:
- changing line 291 to be `wrapOptions: WrapOptions = {}` 
- use extend to do:

```
const defaults = { wrapOptions: true };
const options = extend(true, defaults, wrapOptions, {});
```

Reply:
The interface requires that `WrapOptions`, if provided, has a boolean `background` property,  I don't understand how you're passing `{}`, and if you did the side effect would just be that `background` is falsy.

Reply:
so, the `if (wrapOptions) {` is just checking whether or not `wrapOptions` is `undefined`.

Reply:
updated the code to be less confusing.